stories/topics/proc-aap-aws-backup-and-recovery.adoc

```diff
@@ -17,6 +19,26 @@ Follow the steps below to ensure a smooth backup and recovery process.
 It is important to take a manual snapshot over relying on the automated ones created by an instance as automated snapshots are attached to the lifecycle of the RDS instance. 
 You cannot use an automated snapshot for recovery in the event that your deployment is lost.
 
+Create an On-Demand Backup For the EFS of Your Existing Deployment
+
+.Procedure
+. In the AWS console, navigate to menu:AWS Backup[].
+. From the side menu, navigate to *Dashboard*
+. Click btn:[Create on-demand backup].
+. Under the *Resource Type* dropdown select *EFS*
+. Under the *File sytem ID* dropdown choose the File system ID attached to your AAP Deployment
+. To find the filesystem ID of the deployment
+..  Navigate to menu:CloudFormation[] in the AWS Console and select your deployment stack.
+.. Select the *Resources* tab and expand the *storage* section on the resource tree.
+.. Note the physical ID of the filesystem. 
+The format should be similar to `fs-044fce19b58a6d8f5`.
+. Under the *Backup vault* dropdown select the backup vault you would like to use. Aditional information on creating and configuring backup vault can be found https://docs.aws.amazon.com/aws-backup/latest/devguide/creating-a-vault.html[*here*].
+. At the bottom of the page click btn:[Create on-demand backup]
```

Comment:
You shouldn't need to say where on the page it is. Click btn:[Create on-demand backup].